locales/en-US/server.ftl

```diff
@@ -185,7 +185,9 @@ annotationClearButton =
 annotationCropButton =
     .title = Crop
 annotationSaveButton = Save
+    .title = Save edit
```

Comment:
Unfortunately, for these cases we need to invalidate the entire strings, so you need a new l10n.id

For example

```
annotationSaveButton2 = Save 
    .title = Save edit
```

Reply:
@flodolo I see. I'll make the change, thank you!

Reply:
I think it's nicer to update the string to follow the structure of the string, rather than incrementing integers in the l10n IDs. Something like `annotationSaveEditButton`, rather than `annotationSaveEdit2`.